BackEndApp/tests/utils_tests.py

```diff
@@ -0,0 +1,33 @@
+import os
+import mongomock
+
+from tests.mocks import MockUser
+from authv1.models import Session
+
+
+class TestSession:
+    @mongomock.patch(
+        servers=(
+            (
+                os.getenv("MONGODB_URI"),
+                27017,
+            ),
+        )
+    )
+    def mock_session(self):
```

Comment:
Please change the name of 'mock_session' as you are calling the class

Reply:
made the required changes.